test/common/run-tests

```diff
@@ -154,7 +170,7 @@ class Test:
         nd = f" [ND@{self.machine_id}]" if self.nondestructive else ""
         return f"{self.test_id} {self.command[0]} {self.command[-1]}{cost}{nd}"
 
-    def _print_test(self, print_tap=True, retry_reason="", skip_reason=""):
+    def _print_test(self, print_tap=True, retry_reason="", skip_reason="", todo_reason=None):
```

Comment:
This needs to default to `""`, otherwise you'll be printing "None" in line 205. You could of course also print `todo_reason or ""` below, but then please do the same with skip_reason as well?

Reply:
See [failures like this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18045-20221219-120825-54457608-ubuntu-stable/log.html#154):
![image](https://user-images.githubusercontent.com/200109/208431093-744358ba-adb2-48c3-ad26-efc89069fbba.png)

This both illustrates the `None` issue, and also that this somehow breaks the handling of "SKIP" tests.

Reply:
I [fixed that](https://github.com/cockpit-project/cockpit/pull/18039/commits/52d5b3074cc30e15759f82e59abc084f623dcd71) in #18039. But it still appears as failed, -ish:

```
$ TEST_OS=fedora-36 test/common/run-tests --no-retry-fail --test-dir test/verify TestStorageResize.testResizeLuks
# Result testResizeLuks (__main__.TestStorageResize.testResizeLuks) failed
# 1 TEST FAILED [87s on toolbox]
not ok 1 test/verify/check-storage-resize TestStorageResize.testResizeLuks # SKIP Known issue #1739

# TESTS PASSED [87s on toolbox, 1 parallel tests, 0 serial tests: ]
```
I think it's supposed to be `ok`? Running out of time now, will look tomorrow morning or possibly this evening.

Reply:
Both fixed in a force-push.